guides/introduction/quick_start_guide.md

```diff
@@ -101,6 +101,12 @@ Now that your MeiliSearch server is up and running, you should be able to commun
 
 Communication to the server is done through a [RESTful API](/references/README.md) or one of our [SDKs](/resources/sdks.md).
 
+::: note
+
+For a better experience, we provide a [Postman collection](/resources/postman_collection.md).
+
+:::
+
```

Comment:
You'r right. Maybe, until we find a better way. you could just add it without title: 

```
And additional resources here:

* SDKs
* FAQ
* Postman collection: It can be tedious to re-write every route when wanting to try out an API. Try out MeiliSearch with our collection in Postman. 
```

Reply:
@bidoubiwa done